yarn.lock

```diff
@@ -2499,10 +2563,10 @@ jasmine@^3.1.0:
     glob "^7.1.4"
     jasmine-core "~3.5.0"
 
-jquery@^3.4.0:
-  version "3.4.1"
-  resolved "https://registry.yarnpkg.com/jquery/-/jquery-3.4.1.tgz#714f1f8d9dde4bdfa55764ba37ef214630d80ef2"
-  integrity sha512-36+AdBzCL+y6qjw5Tx7HgzeGCzC81MDDgaUP8ld2zhx58HdqXGoBd+tHdrBMiyjGQs0Hxs/MLZTu/eHNJJuWPw==
+jquery@^1.8.0:
+  version "1.12.4"
```

Comment:
sounds like this is what broke things....

before we were truly on 1.8, and going to 1.12 had the issue with drag and drop.

the weird thing is that if I go down to 1.8 for dependencies and build, i get some odd errors.